### ISSUE TITLE
Crops can still be broken

### ISSUE THREAD

**Issue by 0zlw** [MyEssentials/MyTown2#418]
Fix for crops hasn't seemed to stop them from being broken. Crops will not break when you walk on them however will break when you jump up and down on them in a claimed area.

Tested on VanillaPlus v1.0.0

MyEssentialsCore 100
MyPermissions 47
MyTown2 312

**Comment by joserobjr** [comment]
I just tried to reproduce on a pure and it protected correctly..

Tested on a pure forge server v1.7.10-10.13.4.1566
MyEssentials core dev-100 https://ci.mineguild.net/job/MyEssentials-Core/100/
MyTown2 dev-312 https://ci.mineguild.net/job/MyTown2/312/
MyPermissions dev-47 https://ci.mineguild.net/job/MyPermissions/47/

https://youtu.be/cmspcdcz2PQ

What's VanillaPlus? This mod?
http://www.minecraftforum.net/forums/mapping-and-modding/minecraft-mods/1289132-vanillaplus-v1-5-1-brings-what-should-be-in

**Comment by 0zlw** [comment]
VanillaPlus Modpack, running kCauldron.

**Comment by 0zlw** [comment]
Just watched your video, not getting any sort of message to do with jumping around. kCauldron b144, using forge 1481

**Comment by joserobjr** [comment]
Confirmed on KCauldron-1.7.10-1481.144

**Comment by krwminer** [comment]
Thinking about this last night this was my guess, that you were testing on kcauldron. We should double check the class transformer ASM modification for kcauldron, I bet that is the source of the problem.

**Comment by joserobjr** [comment]
I didn't check on kcauldron when I initially created it but I did after the @0zlw report, I changed the patch strategy to apply it immediately after the return keyword from that cancel it when the `mobGriefing` gameRule is enabled.

Before it was waiting for an `ALOAD 1` (access to argument 1 which is World, `p_149746_1_`) to apply it before it fills the stack with data to invoke `func_147449_b`. This caused an incompatibility on cauldron because it also patches the same location that I did before, so the first `ALOAD 1` happens when it starts loading data to fire an `EntityInteractEvent` on Bukkit side to get a Bukkit's `Block` object.

Now, that it will apply on the beginning first frame that comes after the first return, so it will be the first thing that will happen after the `mobGriefing` code, exactly as I mentioned before (https://github.com/MyEssentials/MyTown2/issues/139#issuecomment-158853527) I patched in a different way to attempt to increase compatibily but I was worng, the fist idea was fine.

When I test I dump a binary version of the modified file, I'll host it temporarily on my Dropbox to show the result of the patch both on pure forge and on kCauldron environment in case you want to decompile and check the result yourself.

To make this dump just add a `FileOutputStream` temporarily to save the file to the disk, like I showed on this example: https://github.com/joserobjr/Example-Clickable-Sign/blob/coremod-with-nbt/src/main/java/com/example/examplemod/SignClassTransformer.java#L232

The srg method name is `func_149746_a`.
https://www.dropbox.com/sh/rgege6qh2vrufyo/AABLDssq3W7-K_0YzBvYBvxqa?dl=0
